contracts/contract-webauthn-secp256r1/src/lib.rs

```diff
@@ -219,20 +188,20 @@ impl CustomAccountInterface for Contract {
         signature: Signature,
         auth_contexts: Vec<Context>,
     ) -> Result<(), Error> {
-        // Only the sudo signer can `add_sig`, `rm_sig`, `resudo` and `upgrade`
+        // Only the super signer can `add_sig`, `rm_sig`, `re_super` and `upgrade`
         for context in auth_contexts.iter() {
             match context {
                 Context::Contract(c) => {
                     if c.contract == env.current_contract_address()
                         && (c.fn_name == Symbol::new(&env, "add_sig")
```

Comment:
nit: Prefer `symbol_short!` for clarity and compile-time check that your symbols are indeed short.